spec/models/user_spec.rb

```diff
@@ -112,4 +112,21 @@
       user.ac?.should eq(true)
     end
   end
+
+  describe '#all_emails' do
+    it 'contains primary email' do
+      user.all_emails.should include(user.email)
+    end
+    it 'contains confirmed additional emails' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.